pkgs/applications/office/beancount/default.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchhg, pkgs, python, pythonPackages }:
```

Comment:
please don't use python here. You can just use pythonPackages.python to get the interpreter.